### PR TITLE
Update to version 0.26.0.

### PR DESCRIPTION
**Pull request by nihohit**

### CODE REVIEW

File: redis/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,125 @@
+### 0.6.0 (2024-07-26)
```
Comment (jaymell) [inline redis/CHANGELOG.md:1]:
```suggestion
### 0.26.0 (2024-07-26)
```

File: redis/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+* Add tests for username+password authentication. [1157](https://github.com/redis-rs/redis-rs/pull/1157)
+* Improve PushManager tests in sync connection ([1100](https://github.com/redis-rs/redis-rs/pull/1100) @altanozlu)
+* Fix issues that prevented cluster tests from running concurrently. [1130](https://github.com/redis-rs/redis-rs/pull/1130)
+* Fix issue in cluster tests. [1139](https://github.com/redis-rs/redis-rs/pull/1139)
+* Remove redundant call. [1112](https://github.com/redis-rs/redis-rs/pull/1112)
+* Fix clippy warnings [#1180](https://github.com/redis-rs/redis-rs/pull/1180)
+* Wrap tests with modules. [1084](https://github.com/redis-rs/redis-rs/pull/1084)
+* Add missing module skips. [#1083](https://github.com/redis-rs/redis-rs/pull/1083)
+* Add vscode settings to gitignore. [1085](https://github.com/redis-rs/redis-rs/pull/1085)
+* Bump serde_json from 1.0.115 to 1.0.116 [#1177](https://github.com/redis-rs/redis-rs/pull/1177)
```
Comment (jaymell) [inline redis/CHANGELOG.md:75]: I would say no obligation to list all the dependabot PRs here.

File: redis/CHANGELOG.md [hunk 3]
```diff
@@ -83,7 +205,7 @@
 * Revert redis-test versioning changes ([#993](https://github.com/redis-rs/redis-rs/pull/993))
 * Tests: Add retries to test cluster creation ([#994](https://github.com/redis-rs/redis-rs/pull/994))
 * Fix sync cluster behavior with transactions. ([#983](https://github.com/redis-rs/redis-rs/pull/983))
-* Sync Pub/Sub - cache received pub/sub messages. ([#910](https://github.com/redis-rs/redis-rs/pull/910))
+* Sync Pub/Sub * cache received pub/sub messages. ([#910](https://github.com/redis-rs/redis-rs/pull/910))
```
Comment (jaymell) [inline redis/CHANGELOG.md:208]: Were these (and below) changes intentional?

Comment (nihohit) [reply redis/CHANGELOG.md:208]: 🤦

**Pull request merged**